### ISSUE TITLE
During handling of the server exception, another exception occurred in sdk

### ISSUE THREAD

**Issue by del-zhenwu** [milvus-io/milvus#4172]
**Please state your issue using the following template and, most importantly, in English.**

**Describe the bug**
```
test_index.py::TestIndexBase::test_create_index_collection_not_existed 
-------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------
--- Logging error ---
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/milvus/client/grpc_handler.py", line 44, in handler
    return func(self, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/milvus/client/grpc_handler.py", line 401, in create_index
    raise BaseError(status.error_code, status.reason)
milvus.client.exceptions.BaseError: <BaseError: (code=1, message=Collection test_index_4n81r8fV not found.)>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.6/logging/__init__.py", line 994, in emit
    msg = self.format(record)
  File "/usr/lib/python3.6/logging/__init__.py", line 840, in format
    return fmt.format(record)
  File "/usr/local/lib/python3.6/dist-packages/_pytest/logging.py", line 71, in format
    return super(ColoredLevelFormatter, self).format(record)
  File "/usr/lib/python3.6/logging/__init__.py", line 577, in format
    record.message = record.getMessage()
  File "/usr/lib/python3.6/logging/__init__.py", line 338, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting

```
**Steps/Code to reproduce behavior**
Follow this [guide](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports) to craft a minimal bug report. This helps us reproduce the issue you're having and resolve the issue more quickly.

**Expected behavior**

**Method of installation**
- [ ] Docker/cpu
- [ ] Docker/gpu
- [ ] Build from source

**Environment details**
- Hardware/Softwars conditions (OS, CPU, GPU, Memory)

- Milvus version (master or released version)

**Configuration file**
Settings you made in `server_config.yaml` or `milvus.yaml`

```yaml
paste-file-content-here
```

**Screenshots**

**Additional context**

**Comment by del-zhenwu** [comment]
```
self = <test_insert.TestInsertAsync object at 0x7fc8e6f469e8>, connect = <milvus.client.stub.Milvus object at 0x7fc9017a3908>

    def test_insert_async_invalid_params(self, connect):
        '''
        target: test insert vectors with different length of vectors
        method: set different vectors as insert method params
        expected: length of ids is equal to the length of vectors
        '''
        collection_new = gen_unique_str()
>       future = connect.bulk_insert(collection_new, default_entities, _async=True)

entity/test_insert.py:776: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
/usr/local/lib/python3.6/dist-packages/milvus/client/stub.py:36: in inner
    return func(self, *args, **kwargs)
/usr/local/lib/python3.6/dist-packages/milvus/client/stub.py:358: in bulk_insert
    info = self.get_collection_info(collection_name)
/usr/local/lib/python3.6/dist-packages/milvus/client/stub.py:36: in inner
    return func(self, *args, **kwargs)
/usr/local/lib/python3.6/dist-packages/milvus/client/stub.py:238: in get_collection_info
    return handler.get_collection_info(collection_name, timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <milvus.client.grpc_handler.GrpcHandler object at 0x7fc9017d2710>, args = ('test_5Dq1wlKP', 30), kwargs = {}
record_dict = {'API start': '2020-11-04 10:47:40.800759', 'RPC start': '2020-11-04 10:47:40.800959'}

    def handler(self, *args, **kwargs):
        record_dict = {}
        try:
            record_dict["API start"] = str(datetime.datetime.now())
            if getattr(self, '_pre_ping'):
                self.ping()
            record_dict["RPC start"] = str(datetime.datetime.now())
            return func(self, *args, **kwargs)
        except BaseError as e:
            LOGGER.error("Error: {}", str(e))
            if e.code == Status.ILLEGAL_COLLECTION_NAME:
                raise IllegalCollectionNameException(e.code, e.message)
            if e.code == Status.COLLECTION_NOT_EXISTS:
>               raise CollectionNotExistException(e.code, e.message)
E               milvus.client.exceptions.CollectionNotExistException: <CollectionNotExistException: (code=4, message=Collection not exist: test_5Dq1wlKP)>

/usr/local/lib/python3.6/dist-packages/milvus/client/grpc_handler.py:50: CollectionNotExistException

```

**Comment by BossZou** [comment]
This bug has been fix at pymilvus-test 0.5.2+. You are recommended to use the latest version .
